### ISSUE TITLE
Chapter 6 Example Code Typos

### ISSUE THREAD

**Issue by chadfawcett** [EthanRBrown/web-development-with-node-and-express#52]
Hello @EthanRBrown 

I am really enjoying the book so far. I was reading through the example code snippets for the **Providing an API** section of Chapter 6 when I came across a couple typos (or at least what I thought may be typos). I wasn't sure if this was an appropriate place to bring these to your attention as they aren't part of the Meadowlark app.

First one I noticed was on page 66 (hard copy) in the first paragraph of the **Providing an API** section. It says

> Examples 6-11 and 6-12 use the following "products" array...

I wasn't sure if this was supposed to say **"tours"** as that is what the array is called or if you were just referring to it as products as tours are a product a tourism company provides.

Secondly, in **Example 6-12** where you format for `text/plain` you are sending the `toursXml` string and not the `toursText` string.

Apologies in advance if these have already been fixed in the eBooks. I just purchased the hard copy a couple days ago so assumed it was a recent version.

**Comment by EthanRBrown** [comment]
Hi, Chad, I'm glad you're enjoying the book!  Yes, the "product" language was an artifact of the prototype example (before I settled on the whole Meadowlark conceit).  I'll get that corrected...also the `toursXml` string...that may have been fixed, but I will have to check to make sure.  There is a place on the O'Reilly website to submit official errata (which I'm behind on; it was on my holiday break to-do list, but time is so short!):

http://www.oreilly.com/catalog/errata.csp?isbn=0636920032977

Thanks again for the corrections!
